driver/config/config.go

```diff
@@ -734,10 +734,6 @@ func (p *Config) ParseURIOrFail(key string) *url.URL {
 		p.l.WithError(errors.WithStack(err)).
 			Fatalf("Configuration value from key %s is not a valid URL: %s", key, p.p.String(key))
 	}
-	if parsed.Scheme == "" {
```

Comment:
Please add a new method that is called for example `ParseAbsoluteOrRelativeURIOrFail` and use it in the places where you want to use it :)

Reply:
Done.